recipes-devtools/python/python3-labgrid_git.bb

```diff
@@ -31,13 +31,13 @@ SRC_URI = " \
     file://environment \
     "
 
-SRCREV = "2437f0f3bc09f24a6af660e4ee26c1141184ce9f"
+SRCREV = "b6fee448c41771fc5fee6c41fd2d6f4904549f19"
 S = "${WORKDIR}/git"
 
 DEPENDS += "python3-setuptools-scm-native"
 DEPENDS += "python3-pytest-runner-native"
 
-inherit setuptools3 systemd
+inherit python_poetry_core systemd
```

Comment:
This does not look correct, we're not using the [Poetry Core](https://python-poetry.org/) build system.

Reply:
Probably `python_setuptools_build_meta` instead?

Reply:
You could be correct here, I went the professional route of googling `pyproject.toml bitbake` and copying the first result from stackoverflow. I will test your suggestion and update this PR if it works.

Reply:
Better search the [Yocto Project Documentation](https://docs.yoctoproject.org/) first 😉